appengine/endpoints/clients/echo_client.rb

```diff
@@ -49,7 +50,7 @@
 options[:message] = "Hello echo message" unless options[:message]
 
 url = "#{options[:host]}/echo?key=#{options[:api_key]}"
-body = { message: options[:message] }.to_json
+body = JSON.generate(message: options[:message])
```

Comment:
Oh, my mistake. The documentation does mention #to_json:
http://ruby-doc.org/stdlib-2.0.0/libdoc/json/rdoc/JSON.html#module-JSON-label-Generating+JSON

I'll revert this.